webapp/src/pages/Settings.tsx

```diff
@@ -31,48 +82,625 @@ const Settings: React.FC = () => {
 
   const resultingConfig = { ...data, ...partialConfig };
 
-  const switchNullOrDefault = (field: keyof AzimuthConfig) => (
+  const divider = <Divider sx={{ marginY: 1 }} />;
+
+  const displaySectionTitle = (section: string) => (
+    <Box sx={{ m: 1, width: "100px" }}>
+      <Typography textTransform="capitalize" variant="subtitle2">
+        {section}
+      </Typography>
+    </Box>
+  );
+
+  const displayToggleSectionTitle = (
+    config: keyof AzimuthConfig,
+    section: string = config
+  ) => (
+    <Box display="flex" flexDirection="row" marginLeft={1.5}>
+      <Typography textTransform="capitalize" variant="subtitle2">
+        {section}
+      </Typography>
+      {switchNullOrDefault(config, Boolean(resultingConfig[config]))}
+    </Box>
+  );
+
+  const displayPostprocessorToggleSection = (pipeline: PipelineDefinition) => (
+    <Box display="flex" flexDirection="row" padding={1}>
+      <Typography variant="subtitle2">Postprocessors</Typography>
+      <Checkbox
+        size="small"
+        sx={{ paddingTop: 0.5 }}
+        checked={Boolean(pipeline.postprocessors)}
+        disabled={isError || isFetching}
+        onChange={(...[, checked]) => {
+          setPartialConfig({
+            ...partialConfig,
+            pipelines: _.unionBy(
+              [
+                {
+                  ...pipeline,
+                  postprocessors: checked
+                    ? _.find(data?.pipelines, ["name", pipeline.name])
+                        ?.postprocessors ?? []
+                    : null,
+                },
+              ],
+              _.has(partialConfig, "pipelines")
+                ? partialConfig.pipelines
+                : resultingConfig.pipelines,
```

Comment:
For the same reason:
```suggestion
              resultingConfig.pipelines,
```

Reply:
As we discussed, I have added the suggestion.